about.html

```diff
@@ -0,0 +1,214 @@
+                    the
+                    creative Commons affiliate network in a different country every two year. This year, we hope to
+                    expand our
+                    inviation list including organisations and individuals who want to work with us on shared projects
+                    that advance the cause of the Commons free culture and open knowledge. So if you're active and
+                    engaged in the worlds of open content and knowledge - free software advocates, Wikipedia, Open
+                    knowledge, galleries, libraries, museums, archives, governments and foundatiosn, lawyers, and
+                    activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons
+                    together
+                </div>
+                <div class="para_about">
+                    The International Summit brings together the community of experts, academics, and activisits who comprisw
+                    the creative Commons affiliate network in a different country every two year. This year, we hope to
+                    expand our inviation list including organisations and individuals. 
+                </div>
+                <div class="about-contact">
+                    Please contact us per Email for any further questions about Coding Minds Summit  2015
+                    <br>
+                    <a class="summit_mail" href="#">codingmindsint2015@cminternational.org</a>
+                </div>
+
+            </div>
+
+        </div>
+
+        <div class="logo_section">
+            <h2 class="logo_head">
+                Coding Minds International Summit 2015 Logo
+            </h2>
+            <hr class="about_them">
+           
+            <p class="logo_des">
+                The logo of Coding Minds International was decide through the logo competion from 8.June to 7.july
+            </p>
+
+            <p class="logo_desk">
+                The logo of Coding Minds International was decide through the logo competion from 8.June to 7.july.
+                After online public voting and Selection Communities judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimised the prize-winning-logo for this year's summit as following:
+            </p>
+
+            <div class="logo_container">
+
+                <div class="log2_cont">
+
+                    <div class="log2">
+                        <img class="log2_img" src="images/logo.png" alt="conference logo">
+                        <div class="log2_title">
+                            <div class="log2_title1">CM</div>
+                            <div class="log2_title2">Summit 2015</div>
+                        </div>
+                    </div>
+
+                    <div class="logo_date">14th-17th Oct | K'la, Uganda</div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix space `arrangement of elements`  on mobile version like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), I `compare` and `highlight` mistakes below the picture:

![image](https://user-images.githubusercontent.com/63915024/219432464-2741eefb-72bb-4caf-8676-951f3eab295b.png)